daemon/endpoint.go

```diff
@@ -54,10 +54,11 @@ func (h *getEndpoint) Handle(params GetEndpointParams) middleware.Responder {
 }
 
 func (d *Daemon) getEndpointList(params GetEndpointParams) []*models.Endpoint {
+	maxGoroutines := 10
```

Comment:
What is the reasoning behind using '10' here? Was there an empirical measurement done? Before, we have used the number of CPUs.

Reply:
Hey @ianvernon This is just a dummy value and the PR is still in progress. Me and @aanm are working on benchmarking/profiling the `getEndpointList` function to decide upon a value/need of maximum go routines limit for this particular function. We are still experimenting and have not reached upon a conclusion yet, I will update the PR with better description once we are done with our research.

Reply:
Got it, sorry for being a bit early on the review !